Tests/RockLib.Messaging.CloudEvents.Tests/CloudEventExtensionsTests.cs

```diff
@@ -182,8 +183,8 @@ public void SetDataMethod3HappyPath3()
         [Fact(DisplayName = "SetData method 3 throws if cloudEvent is null")]
         public void SetDataMethod3SadPath1()
         {
-            CloudEvent cloudEvent = null;
-            var client = new Client { FirstName = "Brian", LastName = "Friesen" };
+            CloudEvent cloudEvent = null!;
+            var client = new TestClient { FirstName = "Brian", LastName = "Friesen" };
```

Comment:
Bad mojo

```suggestion
            var client = new TestClient { FirstName = "Neil", LastName = "Schiefer" };
```

Reply:
Hey now...

Reply:
I mean, how do you know that @NeilSchiefer won't get thrown under the bus at some point in the future? It'd be better to not use any real name at all.

Reply:
Though it _is_ neat to know that I've become Voldemort. 😆

Reply:
haha.  Could happen.

Reply:
updated to use generic names.